apps/web/src/components/settings/RegistrationForm/RegisterFormSettings.tsx

```diff
@@ -0,0 +1,1034 @@
+"use client";
+import { useForm } from "react-hook-form";
+import {
+	Form,
+	FormItem,
+	FormControl,
+	FormDescription,
+	FormMessage,
+	FormLabel,
+	FormField,
+} from "@/components/shadcn/ui/form";
+import {
+	Select,
+	SelectContent,
+	SelectItem,
+	SelectTrigger,
+	SelectValue,
+	SelectGroup,
+} from "@/components/shadcn/ui/select";
+import { Input } from "@/components/shadcn/ui/input";
+import { Button } from "@/components/shadcn/ui/button";
+import { z } from "zod";
+import { RegisterFormValidator } from "@/validators/shared/RegisterForm";
+import { zodResolver } from "@hookform/resolvers/zod";
+import FormGroupWrapper from "@/components/registration/FormGroupWrapper";
+import { Checkbox } from "@/components/shadcn/ui/checkbox";
+import c, { schools, majors } from "config";
+import {
+	Command,
+	CommandEmpty,
+	CommandGroup,
+	CommandInput,
+	CommandItem,
+	CommandList,
+} from "@/components/shadcn/ui/command";
+import {
+	Popover,
+	PopoverContent,
+	PopoverTrigger,
+} from "@/components/shadcn/ui/popover";
+import { Check, ChevronsUpDown } from "lucide-react";
+import { cn } from "@/lib/utils/client/cn";
+import { useEffect, useCallback, useState } from "react";
+import { Textarea } from "@/components/shadcn/ui/textarea";
+import { FileRejection, useDropzone } from "react-dropzone";
+import { put } from "@vercel/blob";
+import { useAction } from "next-safe-action/hook";
+import {
+	modifyRegistrationData,
+	modifyResume,
+} from "@/actions/user-profile-mod";
+import { toast } from "sonner";
+import Link from "next/link";
+import { Loader2 } from "lucide-react";
+
+interface RegistrationData {
```

Comment:
Not sure if this is the last time I will see this, but going forward I won't comment about types anymore.